app/src/main/java/org/jak_linux/dns66/NotificationChannels.java

```diff
@@ -25,8 +25,9 @@
 
     public static void onCreate(Context context) {
         NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O)
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O || notificationManager == null) {
```

Comment:
`context.getSystemService()` may return `null`, so I added a check here. I also added parentheses to match Google's Java style.